docs/faq.rst

```diff
@@ -0,0 +1,126 @@
+.. _faq:
```

Comment:
Now it's hanging loosely without  being discoverable. You need to link to this page somehow. That could be from quickstart, for example, and perhaps even from the start pages of the contributor guide and community?

Reply:
I'll see about adding some of those. It is linked on the landing page, though.

Reply:
The landing page the way you have it at the version of master you pulled doesn't exist anymore. There's the "designed" landing page now, and index.rst is gone. So it will indeed sort of hang loosely and not be discoverable unless we change something.

Reply:
Ok looks like you've linked to it from quickstart. It will only show up when you're on the quickstart page itself, though. Perhaps we need some kind of "About" page that contains FAQ, Vision/Mission. Initially I had thought "Community" is that page, but people wouldn't got to Community to search for FAQs. Perhaps we'll need to review that structure once more

Reply:
I agree that we need to think about that some more. For the moment, I'm happy that it is discoverable, and hence will get linked by search engines etc.